test/drivers/fs.test.ts

```diff
@@ -40,6 +41,40 @@ describe("drivers: fs", () => {
         await ctx.storage.setItem("s1/te..st..js", "ok");
         expect(await ctx.storage.getItem("s1/te..st..js")).toBe("ok");
       });
+
+      it("natively supports maxDepth in getKeys", async () => {
+        await ctx.storage.setItem("depth-test/file0.md", "boop");
+        await ctx.storage.setItem("depth-test/depth0/file1.md", "boop");
+        await ctx.storage.setItem("depth-test/depth0/depth1/file2.md", "boop");
+        await ctx.storage.setItem("depth-test/depth0/depth1/file3.md", "boop");
+
+        const originalStat = await stat(
+          resolve(dir, "depth-test/depth0/file1.md")
+        );
+
+        expect(
+          (
+            await ctx.storage.getKeys(undefined, {
```

Comment:
I think here we might use `ctx.driver.getKeys` to test native `Driver` functionality (without core fallback)